src/common-utils/main.sh

```diff
@@ -311,6 +311,53 @@ install_alpine_packages() {
     PACKAGES_ALREADY_INSTALLED="true"
```

Comment:
Can we bump the minor version for this Feature? 

https://github.com/devcontainers/features/blob/fff9a95f2242a79c83504bd9d4a56ad4b2fa2868/src/common-utils/devcontainer-feature.json#L3